passes/unclosetx/unclosetx.go

```diff
@@ -59,7 +68,7 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			// skip this
 			continue
 		}
-		instrs := analysisutil.NotCalledIn(f, txTyp, methods...)
+		instrs := analysisutil.NotCalledIn(f, txTypeRo, methods...)
```

Comment:
Because the second argument of `NotCalledIn` is a receiver of the methods, it cannot find `txTypeBatch`'s methods.

I think you should call `NotCalledIn` two times for `txTypRo` and `txTypeBatch` and merge theses return values into `instrs`.

Reply:
When I using this:
```
instrs := append(
	analysisutil.NotCalledIn(f, txTypeRo, methods...),
	analysisutil.NotCalledIn(f, txTypeBatch, methods...)...,
)
for _, instr := range instrs {
...
```

The test says:

```
/my/go/path/root/sters/zagane/passes/unclosetx/analysistest.go:446: a/a.go:55:42: unexpected diagnostic: transaction must be closed
```

it points to this case:

![image](https://user-images.githubusercontent.com/1658147/192143661-70d4fbc6-568d-4808-a221-866b346c05c5.png)

I have no idea to solve this test failure. 🤔